lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx

```diff
@@ -49,19 +49,18 @@ class HelpPanelItem extends React.Component {
       flex-shrink: 0;
       width: 64px;
       height: 64px;
-      margin-right: 10px;
       display: table-cell;
+      vertical-align: middle;
+      padding-left: 25px;
     `;
     const CompassIcon = styled(Icon)`
```

Comment:
I think this is a copy-paste-but-didn't-rename-name, little confusing to see compassicon/compasswrapper when it's got nothing to do with the compass 😆

Reply:
Yes! My bad whoops 😆